superset-frontend/src/components/Chart/DrillBy/DrillByModal.tsx

```diff
@@ -106,6 +118,12 @@ export default function DrillByModal({
   dataset,
 }: DrillByModalProps) {
   const theme = useTheme();
+  const [chartDataResult, setChartDataResult] = useState<QueryData[]>();
+  const [showChart, setShowChart] = useState(true);
```

Comment:
Can we use ChartType enum instead of a boolean here? I think it will be easier to understand the intention of showing table instead of just NOT showing the chart 🙂